pilot/pkg/networking/core/cluster_tls.go

```diff
@@ -265,7 +267,7 @@ func constructUpstreamTLS(opts *buildClusterOpts, tls *networking.ClientTLSSetti
 		}
 	}
 
-	applyTLSDefaults(tlsContext, opts.mesh.GetTlsDefaults())
```

Comment:
Naming nit: what's the logic behind changing `xxxTlsDefaults` to `xxxTlsConfig`?